router/sleeve.go

```diff
@@ -363,6 +370,8 @@ func (sleeve *SleeveOverlay) MakeForwarder(params ForwarderParams) (OverlayForwa
 		specialChan:      specialChan,
 		confirmedChan:    confirmedChan,
 		finishedChan:     finishedChan,
+		establishedChan:  make(chan struct{}),
+		errorChan:        make(chan error, 1),
```

Comment:
It's odd that these two channels are initialised without the use of intermediary variables as with all the other chans in this struct